doc/api.md

```diff
@@ -212,6 +230,10 @@ For permanent storage, use `npc.data.*` functions.
 * `npc.exec.var.get(luaentity, key_name)`: Returns the value of a given key_name
 * `npc.exec.var.set(luaentity, key_name, new_value)`: Update a value in the execution context
 * `npc.exec.var.remove(luaentity, key_name)`: Remove a variable from the execution context
+* `npc.data.put(luaentity, key_name, value, readonly)`: This function adds a value to the execution context of the current process
```

Comment:
This says "execution context of the current process". It should say, "permanent data storage in the Lua entity" or similar